src/locale/locales/en-US.json

```diff
@@ -25,6 +25,18 @@
       "integer": "Integer",
       "percentage": "Percentage"
     },
+    "accrualPeriodicity": {
+      "label": "Accrual periodicity",
+      "help": "The frequency with which the KPI are expected to be updated with progress data.",
```

Comment:
Blir "with witch" => "at which" og "are" => "is" mer riktig?

Reply:
Trenger kanskje ikke være korrekt engelsk selv om det er stjålet fra Dublin Core 😓 
> Accrual Periodicity
> The frequency with which items are added to a collection.
https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/accrualPeriodicity